c2corg_ui/static/js/documentediting.js

```diff
@@ -231,8 +230,18 @@ app.DocumentEditingController.prototype.successRead_ = function(response) {
   this.scope_[this.modelName_] = data;
 
   if (this.modelName_ === 'outing') {
-    this.scope_['outing'] = this.formatOuting_(this.scope_['outing']);
-    this.differentDates = app.utils.areDifferentDates(this.scope_['outing']['date_start'], this.scope_['outing']['date_end']);
+    // check if user has right to edit -> is one of the associated users
+    var hasRight = app.UserController.prototype.hasEditRightsOuting(this.scope_['outing']['associations']['users']);
```

Comment:
I think you would better put the logics in an Angular service and directly call the service here instead of going through the UserController prototype.